lwesp/src/lwesp/lwesp_int.c

```diff
@@ -1977,10 +1977,18 @@ lwespi_initiate_cmd(lwesp_msg_t* msg) {
 #if LWESP_CFG_MODE_STATION
         case LWESP_CMD_WIFI_CWJAP: { /* Try to join to access point */
             AT_PORT_SEND_BEGIN_AT();
-            AT_PORT_SEND_CONST_STR("+CWJAP=");
+            if (msg->msg.sta_join.mesh_id){
```

Comment:
I wonder if using the same command (`LWESP_CMD_WIFI_CWJAP`) for joining a normal network vs a mesh one is a good idea. Initially I though that you did it so that we don't need to modify the FW to support mesh, but then I saw there's a different API to join a mesh.

What are your thoughts about having a specific command for that?